### DISCUSSION TITLE
티스토리 기본 리스트 적용시 스타일이 적용 안되는데 혹시 의도된 것인가요?

### DISCUSSION THREAD

**Discussion by progrunman** [pronist/hELLO#109] [Q&A] [answered]
안녕하세요. 좋은 스킨을 알게되어 잘 쓰고 있습니다.
글쓰기에서 리스트 삽입 후, 리스트 2번째 스타일 적용시에도 전부 첫번째 스타일로 고정되는데 혹시 의도된 것인지 궁금합니다.

1단에 첫번째스타일, 2단에 두번째스타일을 자주 섞어쓰는데
글 발행시 모두 첫번째 스타일로 통일되어 버리네요

만약 스킨에서 따로 스타일 변경이 없다면 default 스타일이 그대로 유지되는것이 어떨런지요

**Comment by pronist** [comment, marked as answer]
@progrunman 안녕하세요, 리스트 관련 문의를 주셨군요. 이 부분은 따로 의도를 한 것은 아닙니다만, 추후 패치를 통해 변경을 검토해보도록 하겠습니다.
